src/ops/info.rs

```diff
@@ -1,17 +1,13 @@
 //! The info callable is for printing
 
 use crate::ops::error::{ok, OpResult};
-use crate::project;
-use crate::VERSION_BUILD_REV;
+use slog_scope::info;
 
 /// See the documentation for lorri::cli::Command::Info for more
 /// details.
-pub fn main(project: project::Project) -> OpResult {
-    println!("lorri version: {}", VERSION_BUILD_REV);
-    println!("Lorri Project Configuration");
-    println!();
-
-    println!("expression: {}", project.nix_file);
-
+pub fn main() -> OpResult {
+    // lorri version and root shell.nix file are attached to the logger as key-value pairs, so they
+    // are part of every log line.
+    info!("Hello Nix!");
```

Comment:
Let's revert this change -- the `println` output, I think, is useful as part of UX for the user. If we decide to drop this op, let's delete the whole op... but not sure we should right now.

Reply:
Reverted the change.